src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/help/New-AzureRmRecoveryServicesAsrReplicationProtectedItem.md

```diff
@@ -65,8 +74,24 @@ Starts the replication protected item creation operation for the specified ASR p
 PS C:\>$job = New-AzureRmRecoveryServicesAsrReplicationProtectedItem -ProtectableItem $pi -Name $rpiName -ProtectionContainerMapping $pcm -RecoveryAzureStorageAccountId $RecoveryAzureStorageAccountId -RecoveryResourceGroupId  $RecoveryResourceGroupId -ProcessServer $fabric.fabricSpecificDetails.ProcessServers[0] -Account $fabric.fabricSpecificDetails.RunAsAccounts[0]
 ```
 
+### Example 3
+```
+PS C:\>$job = New-AzureRmRecoveryServicesAsrReplicationProtectedItem -AzureToAzure -AzureToAzureDiskReplicationConfig disk1,disk2 -AzureVmId $vmId
+ -Name <String> -RecoveryVmName "vmName" -ProtectionContainerMapping $pcmMapping
+ -RecoveryResourceGroupId <String>
+```
+
 Starts the replication protected item creation operation for the specified ASR protectable item and returns the ASR job used to track the operation.
```

Comment:
@viverm same comment about the description being moved from the previous example to this one. Also, same comment about the user not being able to copy-paste this example since the multi-line scenario will be broken without a trailing backtick ( ` ) to signal continuation on the next line